torchrl/modules/planners/cem.py

```diff
@@ -13,28 +13,26 @@
 
 
 class CEMPlanner(MPCPlannerBase):
-    """
-    CEMPlanner Module. This class inherits from TensorDictModule.
+    """CEMPlanner Module. 
+
+    Reference: The cross-entropy method for optimization, Botev et al. 2013
 
-    Provided a TensorDict, this module will perform a CEM planning step.
+    This module will perform a CEM planning step when given a TensorDict containing initial states.
     The CEM planning step is performed by sampling actions from a Gaussian distribution with zero mean and unit variance.
-    The actions are then used to perform a rollout in the environment.
-    The rewards are then used to update the mean and standard deviation of the Gaussian distribution.
-    The mean and standard deviation of the Gaussian distribution are then used to sample actions for the next planning step.
+    The sampled actions are then used to perform a rollout in the environment. The cumulative rewards obtained with the rollout is then 
+    ranked. We select the top-k episodes and use their actions to update the mean and standard deviation of the actions distribution.
     The CEM planning step is repeated for a specified number of steps.
-    At the end, we recover the best action which is the one that maximizes the reward given a planning horizon.
+
+    A call to the module returns the actions that empirically maximised the returns given a planning horizon
 
     Args:
-        env (Environment): The environment to perform the planning step on (Can be ModelBasedEnv or EnvBase).
-        planning_horizon (int): The number of steps to perform the planning step for.
-        optim_steps (int): The number of steps to perform the MPC planning step for.
-        num_candidates (int): The number of candidates to sample from the Gaussian distribution.
+        env (EnvBase): The environment to perform the planning step on (can be ```ModelBasedEnv``` or ```EnvBase```).
```

Comment:
not sure we want three ` here, that will create a block no? Just one would suffice I think